themes/scss/widgets/layout.scss

```diff
@@ -30,8 +27,7 @@
 
 @include media-query-mobile() {
   .CDB-Widget-contentSpaced--sideMargins {
-    margin-right: $sMargin-section/3;
-    margin-left: $sMargin-section/3;
+    margin-right: $sMargin-section / 3;
```

Comment:
Name of variable `sMargin-section` should be written in all lowercase letters with hyphens instead of underscores<br>Properties should be ordered margin-left, margin-right